utils/mstpctl-utils-functions.sh

```diff
@@ -6,6 +6,7 @@ mstpctl_parse_ports()
     # For compatibility: the 'all' option.
     case $1 in
       all)
+	# shellcheck disable=SC2121
 	shift &&
 	set regex eth.\* em.\* 'p[0-9].*' noregex "$@"
```

Comment:
yep ; shellcheck will not complain if i added the 2 dashes ; wasn't sure, so i just silenced shellcheck